### PR TITLE
Documents changes to accounts included relationships and new holdings total APIs

### PR DESCRIPTION
**Pull request by antonivanopoulos**
https://www.notion.so/sixpark/Six-Park-Update-API-documentation-69bc4ff7a1c1463f9625f705cf980aa3

### CODE REVIEW

File: source/includes/_holding_totals.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+          }
+        }
+      }
+    },
+    "..."
+  ]
+}
+```
+
+_Retrieve_ a collection of the resource owner's accounts.
```
Comment (adriancb) [inline source/includes/_holding_totals.md]: Copy/🍝 .

File: source/includes/_holding_totals.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+Not Applicable
+
+### Summary
+
+Configuration | Value | Description
+--------- | ------- | -----------
+authenticated | resource owner | Access is granted via the resource owner's access token
+paginated | no |
+
+### 201 HTTP status code properties
```
Comment (adriancb) [inline source/includes/_holding_totals.md]: I'm not sure why these are classified as `201s`... 🤔  There seems to be a few (unrelated to your changes).

File: source/includes/_holding_totals.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+Property | Type | Description
+--------- | ----------- | -----------
+id | string | Unique identifier for the holdings total object
+as_at_date | string | The date represented by this snapshot of the acounts performance
```
Comment (adriancb) [inline source/includes/_holding_totals.md]: We should settle on either `performance`, `holding_totals` or `value of the account` and remain consistent throughout.

File: source/includes/_accounts.md [hunk 4]
```diff
@@ -201,14 +300,85 @@ curl "https://app.sixpark.com.au/api/v1/users/:user_id/accounts/:account_id"
...
+      "user": "https://...",
+      "self": "https://..."
+    }
+  },
+  "included": [
+    {
+      "id": "ed83e246-c1ce-499a-9ed7-28c38fac1e7a",
+      "type": "holdings_total",
+      "attributes": {
+        "value": 868151.31,
```
Comment (adriancb) [inline source/includes/_accounts.md:326]: Where do you think the documentation for these attributes will exist?

**Review by adriancb — CHANGES_REQUESTED** [review]
Nice! Are you thinking there will be a separate PR for the `portfolio` endpoint?

File: source/includes/_holding_totals.md (hunk 2 above)
Comment (antonivanopoulos) [reply source/includes/_holding_totals.md]: Cool, I noticed this as well, so I might just adjust all of them where they're off.

File: source/includes/_accounts.md (hunk 4 above)
Comment (antonivanopoulos) [reply source/includes/_accounts.md:326]: I might looks at doing it in the `_holdings_totals` page itself, and have an intro that details the full model properties, similar to https://woocommerce.github.io/woocommerce-rest-api-docs/#coupon-properties.

Would be handy to do that for holdings as well, given they don't have an API of their own.

**Comment by antonivanopoulos** [comment]
>Are you thinking there will be a separate PR for the portfolio endpoint?

We don't have a portfolio endpoint? At the moment it's just a namespace in the routes that the holdings totals index sits under... Happy to add it back in as a single resource endpoint though.

**Comment by adriancb** [comment]
> > Are you thinking there will be a separate PR for the portfolio endpoint?
> 
> We don't have a portfolio endpoint? At the moment it's just a namespace in the routes that the holdings totals index sits under... Happy to add it back in as a single resource endpoint though.

As in - to remove it from the documentation, etc.

**Comment by antonivanopoulos** [comment]
@adriancb Ah cool, then nah, I ended up pulling that out in the latest changes.

**Review by adriancb — APPROVED** [review]

**Pull request merged**